### ISSUE TITLE
Single-GPU Training

### ISSUE THREAD

**Issue by jimmy-dq** [PeizeSun/TransTrack#4]
Hi,
Thanks for your great work. I want to know that does single-gpu training (i.e., V100) lead to large performance drop?

**Comment by PeizeSun** [comment]
Hi~
We didn't train the model in single-gpu. I guess if training with single-gpu training, appropriately adjusting learning rate, batch size and training epochs could obtain the same result as multiple-gpu training.

**Comment by handworker** [comment]
> Hi~
> We didn't train the model in single-gpu. I guess if training with single-gpu, appropriately adjusting learning rate, batch size and training epochs could obtain the same result as multiple-gpu training.

Thanks for your reply. I will try, hope that the same result can be achieved.

**Comment by sramakrishnan247** [comment]
@PeizeSun 
How long did it take to train TransTrack after the detector was trained on Crowdhuman dataset?

**Comment by PeizeSun** [comment]
@sramakrishnan247 It takes about 1 hour on 8-gpus.

**Comment by jer1ry** [comment]
@jimmy-dq
Hi
Have you successfully trained with a single GPU? Can you tell us about your parameter settings, such as learning rate, batch size and training epochs?Thank you

**Comment by jimmy-dq** [comment]
> @jimmy-dq Hi Have you successfully trained with a single GPU? Can you tell us about your parameter settings, such as learning rate, batch size and training epochs?Thank you

Not yet. I do not try this single-gpu training.

**Comment by jer1ry** [comment]
@jimmy-dq Thank you for your reply.Did you reproduce the code from this paper? Did you get the desired result?
